status/projects.py

```diff
@@ -167,6 +167,15 @@ def project_summary_data(self, row):
             except ValueError:
                 pass
 
+        #Try to fetch a name for the contact field, not just an e-mail
+        if 'contact' in row.value:
+            if 'owner' in row.value['order_details'] and row.value['contact'] == row.value['order_details']['owner']['email']:
+                row.value['contact'] = str(row.value['order_details']['owner']['name']) + ': ' + str(row.value['order_details']['owner']['email'])
```

Comment:
Why is this cast to string? Is it not already a string?

Reply:
You would think so, but I got an error that you can't combine string- and nonetype. 
I've added multiple checks for `None` and still got the same error, so I just made all the fields strings for it to work 🤷‍♀️